easybuild/easyconfigs/v/VMD/VMD-1.9.4a43-fosscuda-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,62 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Authors::   Stephane Thiell <sthiell@stanford.edu>
+##
+name = 'VMD'
+version = '1.9.4a43'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.ks.uiuc.edu/Research/vmd'
+description = """VMD is a molecular visualization program for displaying, animating, and analyzing large biomolecular
+ systems using 3-D graphics and built-in scripting."""
+
+toolchain = {'name': 'fosscuda', 'version': '2019b'}
+
+source_urls = [
+    'https://www.ks.uiuc.edu/Research/vmd/vmd-1.9.4/files/alpha/',
+    'https://www.ks.uiuc.edu/Research/vmd/vmd-%(version)s/files/final',
+    'http://webclu.bio.wzw.tum.de/stride/'
+]
+sources = [
+    'vmd-%(version)s.src.tar.gz',
+    {'filename': 'stride.tar.gz', 'extract_cmd': "tar -C vmd-%(version)s/lib/stride -xf %s"},
+]
+patches = [
+    ('VMD-1.9.3_plugins.patch'),
+    ('VMD-1.9.3_surf_Makefile.patch', 'vmd-%(version)s'),
+    ('VMD-1.9.3_surf_bad_printfs.patch', 'vmd-%(version)s'),
+    ('VMD-1.9.3_stride_Makefile.patch', 'vmd-%(version)s'),
+    ('VMD-1.9.3_stride_MAX_AT_IN_RES.patch', 'vmd-%(version)s'),
+    ('VMD-1.9.4_configure.patch', 'vmd-%(version)s'),
+    ('VMD-1.9.4_extra_colors.patch', 'vmd-%(version)s'),
+]
+checksums = [
+    '84323b2c34db8ce5739372dd6e225ef1fa1dc5c4b82d3810d55923a653b1bdc0',  # vmd-1.9.4a43.src.tar.gz
+    '51a8bc2988bb184bd08216124f61725225bb1a6f563bdf8cd35154cb5d621c1a',  # stride.tar.gz
+    '7779c6ceedb6e58312a09228f142580a5a60bffec917fd8e34f51144ed020e0f',  # VMD-1.9.4_configure.patch
+    '0f45573d4b01133dc88b3238e2286db1d6a54c7cb6a4fc411fb3600aed617149',  # VMD-1.9.4_plugins.patch
```

Comment:
@bedroge the comment suggests it's for 1.9.4, whereas in the patches list you specify 1.9.3?